tests/transport_test.go

```diff
@@ -60,12 +60,21 @@ var _ = Describe("Transport Tests", func() {
 			sec *v1.Secret
 		)
 
+		var stringData map[string]string
 		if credentialed {
```

Comment:
Should be
`if accessKey != "" || secretKey != "" {...}`

Reply:
sure